cypress/e2e/trigger_builder.cy.js

```diff
@@ -315,11 +335,11 @@ describe('Trigger builder tests', () => {
       });
 
       it('has a Hide button to toggle Trigger Source visibility', () => {
-        cy.get('#triggerSource').scrollIntoView().should('be.visible');
-        cy.get('button').contains('Hide source').scrollIntoView().click();
-        cy.get('#triggerSource').should('not.exist');
-        cy.get('button').contains('Show source').scrollIntoView().click();
-        cy.get('#triggerSource').scrollIntoView().should('be.visible');
+        cy.get('#triggerSource', { timeout: 1000 }).scrollIntoView().should('be.visible');
+        cy.get('button').contains('Hide source', { timeout: 1000 }).scrollIntoView().click();
+        cy.get('#triggerSource', { timeout: 1000 }).should('not.exist');
+        cy.get('button').contains('Show source', { timeout: 1000 }).scrollIntoView().click();
+        cy.get('#triggerSource', { timeout: 1000 }).scrollIntoView().should('be.visible');
```

Comment:
if not required, all of those timeouts would only prolong the duration of this already long tests

This apply for the entire PR, check and be sure the timeouts/cy.wait functions are really needed